project/Build.scala

```diff
@@ -112,8 +112,8 @@ object MimaBuild {
              publish := (),
              publishLocal := (),
              publishSigned := (),
-             sbtVersion in Global := "0.13.13", // Should be ThisBuild, but ^^ uses Global (incorrectly)
-             crossSbtVersions := List("0.13.13", "1.0.0-RC3"), // Should be ThisBuild, but Defaults defines it at project level..
+             sbtVersion in Global := "0.13.17", // Should be ThisBuild, but ^^ uses Global (incorrectly)
+             crossSbtVersions := List("0.13.17", "1.1.1"), // Should be ThisBuild, but Defaults defines it at project level..
```

Comment:
As sbt plugins API is not forwards-compatible, I think we should use the latest version of the 1.0.x series here. So we make sure that we do not use features from 1.1.x and break the plugin for 1.0.x users.

Reply:
Because no version of sbt (including 1.0.x releases) is forwards-compatible I suggest we use 0.13.13 and 1.0.0.